### PR TITLE
fix: the reply button UI

### LINKED ISSUE #3655: [feature request] Update ButtonShowReplies component
> This ticket is part of series of 7 tickets concluding a design task focusing on improving design and usability of comments feature. Here you can find the [Figma file](https://www.figma.com/board/GfQPMs7PyC7jbkymDTco2D/CP-%2F-Nested-Comments-Improvements?node-id=0-1&t=7sgMA4ZPYXAMb03A-1) for more context.

### **6/7 - Update ButtonShowReplies component**
---

**Is your feature request related to a problem? Please describe.**
The button is visually very dominant, the border makes it very loud and prominent and it visually clutters the comment space, since it appears under every comment.

image

**Describe the solution you'd like**
A version of the button without the border, similar to how all the other platforms with comments such as youtube or reddit do it. 

image

Note that in the screenshots below there are other changes to the button, namely the button text and the icon that are not in the scope of this issue, they are separated here #xxxx.

image

:hover

image

**Describe alternatives you've considered**
None

**Additional context**
See the milestone

### PR DESCRIPTION
**Pull request by prashik0202**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

_What this PR does_
This PR updates the "Show Replies" button functionality. It enhances the user experience by making the button more intuitive and responsive, ensuring that replies are displayed or hidden correctly when the button is clicked.

## Git Issues

Closes #3655

## Screenshots/Videos

_If useful, provide screenshot or capture to highlight main changes_
Screenshot 2024-06-18 165306

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of our monthly maintainers call (first Monday of the month)

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

### CODE REVIEW

File: packages/components/src/ButtonShowReplies/ButtonShowReplies.tsx [hunk 1]
```diff
@@ -23,11 +23,12 @@ export const ButtonShowReplies = (props: Props) => {
       data-cy={`show-replies`}
       icon={icon}
       onClick={setIsShowReplies}
-      sx={{ alignSelf: 'flex-start' }}
+      sx={{ alignSelf: 'flex-start', border: 'none' }}
       variant="outline"
       small
     >
       {text} {creatorName && ` to ${creatorName}`}
+      {/* {text} */}
```
Comment (benfurber) [inline packages/components/src/ButtonShowReplies/ButtonShowReplies.tsx:31]: I don't think this line should be included.

**Review by benfurber — CHANGES_REQUESTED** [review]
Awesome! Thank you @prashik0202 for your contribution!

File: packages/components/src/ButtonShowReplies/ButtonShowReplies.tsx (hunk 1 above)
Comment (prashik0202) [reply packages/components/src/ButtonShowReplies/ButtonShowReplies.tsx]: sure!

**Comment by benfurber** [comment]
@all-contributors please add @prashik0202 for code

**Pull request merged**